script.js

```diff
@@ -0,0 +1,84 @@
+    fetch('./CountriesData.json')
+        .then(response => response.json())
+        .then(data => {
+            const container = document.querySelector('.countries-grid');  // המיכל שבו תוסיף את כל המדינות
+    
```

Comment:
Please do not use Hebrew comments, it will not support in every device, in addition the code should be self explanatory and understandable without comments